demisto_sdk/commands/common/tools.py

```diff
@@ -1151,6 +1164,12 @@ def find_type(path: str = '', _dict=None, file_type: Optional[str] = None, ignor
     except FileNotFoundError:
         # unable to find the file - hence can't identify it
         return None
+    except ValueError as err:
+        logger.debug(str(err))
+        if ignore_invalid_schema_file:
+            # invalid file schema
+            return None
+        raise
```

Comment:
let's raise err here to not hide the original exception in case the handling goes south

Reply:
done